client/extensions/woocommerce/woocommerce-services/views/shipping-label/style.scss

```diff
@@ -82,6 +79,19 @@
 	}
 }
 
+&.dialog.card.label-details-modal {
+	min-width: 35%;
```

Comment:
There's still a ton of empty space, specially in a widescreen monitor. What about putting here a constant width, like 400px or so? And `width` instead of `min-width`, since all the elements inside the dialog should wrap seamlessly if they're too long (they're just `<li>` with text inside).

Reply:
👍 Made it 460px but can go narrower still if you or anyone thinks it's best. https://cloudup.com/czCgeNmQ2x8